NitroxServer/Communication/Packets/Processors/PinMovedProcessor.cs

```diff
@@ -0,0 +1,13 @@
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+
+namespace NitroxServer.Communication.Packets.Processors;
+
+public class PinMovedProcessor : AuthenticatedPacketProcessor<PinMoved>
```

Comment:
Are these pins just recipes or can they be something else?  If just recipe, let's name it as such.  Just like you did with `RecipePinned` below